aries-site/src/examples/components/table/TableSingleSelectExample.js

```diff
@@ -230,66 +247,24 @@ export const TableSingleSelectExample = () => {
         direction="row-responsive"
       >
         <Box gap="small" direction="column">
-          <Text>Order Name:</Text>
-          <Text>Purchase Order:</Text>
-          <Text>State:</Text>
-          <Text>Service:</Text>
-          <Text>Tenant:</Text>
-          <Text>Contact:</Text>
-          <Text>Order Date:</Text>
+          <Text size="small">Order Name:</Text>
+          <Text size="small">Purchase Order:</Text>
+          <Text size="small">State:</Text>
+          <Text size="small">Service:</Text>
+          <Text size="small">Tenant:</Text>
+          <Text size="small">Contact:</Text>
+          <Text size="small">Order Date:</Text>
         </Box>
         <Box gap="small" direction="column">
-          {pageDetails.orderName}
-          {pageDetails.purchaseOrder}
-          {pageDetails.state}
-          {pageDetails.service}
-          {pageDetails.tenant}
-          {pageDetails.contact.email}
-          {pageDetails.orderDate}
+          <Text size="small">{pageDetails.orderName} </Text>
```

Comment:
Any way to store these data items in an array and map through? instead of having to repeat `<Text size="small">` each time?

Same with the above section